### PR TITLE
Update gsq-roles.ttl

### PR DESCRIPTION
**Pull request by UbuntuScott**
Adding 2 new DES roles
Please review

### CODE REVIEW

File: vocabularies/gsq-roles.ttl [hunk 1]
```diff
@@ -16,6 +16,7 @@
     dcterms:source <https://standards.iso.org/iso/19115/resources/Codelists/gml/CI_RoleCode.xml> ;
     skos:definition "The roles of Agents (People and Organisations) in relation to Entities (Datasets, Samples etc.)"@en ;
     skos:hasTopConcept gsqr:contractor,
+        des-im:manager,
```
Comment (KellyVance) [inline vocabularies/gsq-roles.ttl:19]:
```suggestion
        gsqr:manager,        
```

File: vocabularies/gsq-roles.ttl [hunk 2]
```diff
@@ -34,6 +35,23 @@
     sdo:name "Geological Survey of Queensland" ;
     sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
 
+des-im:administrator
```
Comment (KellyVance) [inline vocabularies/gsq-roles.ttl:38]:
```suggestion
gsqr:administrator
```

File: vocabularies/gsq-roles.ttl [hunk 3]
```diff
@@ -34,6 +35,23 @@
...
+  a skos:Concept ;
+  a dcat:Role ;
+  rdfs:label "Data Administrator" ;
+  skos:broader des-im:manager ;
+  skos:definition "Design, implementation and maintenance of the technology environment that supports/enables the information management lifecycle."@en ;
+  skos:inScheme des-im:DES-IM-Roles ;
+  skos:prefLabel "Data Administrator" .
+
+des-im:manager
```
Comment (KellyVance) [inline vocabularies/gsq-roles.ttl:47]:
```suggestion
gsqr:manager
```

**Review by KellyVance — COMMENTED** [review]
requires prefix corrections, as per suggestions

**Comment by UbuntuScott** [comment]
Des roles updated

**Pull request merged**